README.md

```diff
@@ -131,7 +131,7 @@ plot.show()
 ```
 ## Links to Datasets
 * **Segmenting the C. elegans embryo**
-    * The model was trained on the [WormImage dataset](https://www.wormimage.org/) with manually labelled masks. You can download the segmentation datset [from this link](https://github.com/devoworm/GSoC-2020/raw/master/Pre-trained%20Models%20(DevLearning)/notebooks/embryo_segmentation/data/image_data.tar.xz). 
+    * The dataset was sourced from [3DMMS](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2720-x#Abs1/). You can download the segmentation datset [from this link](https://github.com/devoworm/GSoC-2020/raw/master/Pre-trained%20Models%20(DevLearning)/notebooks/embryo_segmentation/data/image_data.tar.xz). 
```

Comment:
> `You can download the segmentation datset [from this link](https://github.com/devoworm/GSoC-2020/raw/master/Pre-trained%20Models%20(DevLearning)/notebooks/embryo_segmentation/data/image_data.tar.xz).`

This is not needed, this dataset is just a small subet of the WormImage dataset. It is not for the model that I used in devolearn.